raiden/tests/fixtures/blockchain.py

```diff
@@ -226,9 +201,12 @@ def cached_genesis(request, blockchain_type):
 
         genesis_alloc[account_address] = account_alloc
 
+    all_keys = list(private_keys)
```

Comment:
Why use a set at all, the private_keys fixture checks for duplicates:

https://github.com/raiden-network/raiden/blob/master/raiden/tests/fixtures/variables.py#L158